.npmrc

```diff
@@ -1,4 +1,4 @@
-registry=https://registry.npmjs.org
+registry=https://npm.dev.paypalinc.com
```

Comment:
I think this has to be the public npm registry because GitHub actions will fail to download dependencies from a non-public registry.

Reply:
Got it! Thanks Jesse.